<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSearchUser</class>
 <widget class="QWidget" name="AdminSearchUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>甜鱼: 管理员搜索用户界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    border: none;
    border-radius: 4px;
	padding: 4px;
	margin: 4px; 
    background-color: rgb(54, 125, 221);
    color: white;
}
QLineEdit{
	border: 1px solid black;
	border-radius: 4px;
	padding: 4px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>搜索关键字:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="check_uid">
       <property name="text">
        <string>UID</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>搜索内容:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="check_name">
       <property name="text">
        <string>用户名</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>甜鱼二手交易平台</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>管理员搜索用户界面</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="postBtn">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="search_content_text"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
